mlflow/tracking/fluent.py

```diff
@@ -3129,6 +3145,14 @@ def predict(model_input):
         traces = mlflow.search_traces(model_id=mlflow.get_active_model_id(), return_type="list")
         assert len(traces) == 1
     """
+    active_model = _set_active_model(name=name, model_id=model_id)
+    _ACTIVE_MODEL_ID_STATE.set(ActiveModelIdState.USER)
+    return active_model
+
+
+# NB: This function is only intended to be used internally by MLflow to set
+# the active model ID under some specific conditions (e.g. load_model calls)
```

Comment:
Do we need this comment? It's a private which basically means "Don't touch it if you're an MLflow dev".